code/game/gamemodes/objective.dm

```diff
@@ -213,6 +213,12 @@ GLOBAL_LIST_INIT(potential_theft_objectives, (subtypesof(/datum/theft_objective)
 	name = "Debrain"
 	martyr_compatible = 0
 
+/datum/objective/debrain/is_invalid_target(datum/mind/possible_target)
+	if(..() || !possible_target.current.client)
```

Comment:
this removes SSD people from the target pool, is this intended?

Reply:
This was taken of the reroll code for escape for identity, I presumed it was there for a reason, can remove if not needed